### PR TITLE
One fixed section and one added section, README update

### PR DESCRIPTION
**Pull request by Setol21**
Fixed anaconda section and added the first example to DeepXDE section. 🚀 

Don't delete the DeepXDE section issue, it lacks another example. 😄

### CODE REVIEW

**Comment by pescap** [comment]
@Setol21 could you check why Travis CI fails?

File: docs/source/deepxde.rst [hunk 1]
```diff
@@ -1,8 +1,205 @@
 DeepXDE
 =======
 
+
 To do. Explain the theory for inverse problems, and add references.
 
+Solving a Time-independent PDE: 
+-------------------------------
+This mathematical example is commonly known,
```
Comment (pescap) [inline docs/source/deepxde.rst:9]: commonly known: I didn't understand what you meant

File: docs/source/sysadmin.rst [hunk 2]
```diff
@@ -112,25 +112,32 @@ Conda is a package and environment management system, allowing to easily find an
...
+       1. Download the latest version of `Conda <https://www.anaconda.com/products/individual>`_ (copy the link of download)::
+
+            $ curl -O donwload_link
 
-            $ curl -O <url of conda installer>
        2. Run::
 
             $ bash <name of file downloaded>.sh
 
        3. Follow the instructions on the installer screens.
```
Comment (pescap) [inline docs/source/sysadmin.rst:123]:
```suggestion
       3. Follow the instructions on the installer screen.
```

**Comment by pescap** [comment]
Very good job!

**Comment by Setol21** [comment]
Travis says some weird stuff, this is the principal issue:

- error: metadata-generation-failed
- The command "pip install -r requirements.txt" failed and exited with 1 during .

**Pull request closed without merging**